crates/worker/src/partition_processor_manager/mod.rs

```diff
@@ -85,10 +89,13 @@ pub struct PartitionProcessorManager {
     tx: mpsc::Sender<ProcessorsManagerCommand>,
 
     persisted_lsns_rx: Option<watch::Receiver<BTreeMap<PartitionId, Lsn>>>,
+    archived_lsns_tx: HashMap<PartitionId, watch::Sender<Option<Lsn>>>,
```

Comment:
Here we hold the transmit channels to update the individual PPs on successful snapshot archival.

Reply:
Could this be handled purely in the partition processor manager?